ChangeLog.md

```diff
@@ -24,6 +24,8 @@ Major changes:
 
 Behavior changes:
 
+* `stack setup --resolver X` ran in the global project will now
+  put resolver X in the implicit global project config.
```

Comment:
This sounds like I can update the resolver of the global project by running `stack setup --resolver X`.

As I wrote in https://github.com/commercialhaskell/stack/issues/2579#issuecomment-248795488 the behaviour change should also apply to other commands, not just `setup`.